Functions.Templates/Templates/HttpTrigger-CSharp-Isolated/HttpTriggerCSharp.cs

```diff
@@ -1,20 +1,31 @@
 using System.Net;
 using Microsoft.Azure.Functions.Worker;
-using Microsoft.Azure.Functions.Worker.Http;
 using Microsoft.Extensions.Logging;
+#if NetCore
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+#elif NetFramework
+using Microsoft.Azure.Functions.Worker.Http;
+#endif
 
 namespace Company.Function
 {
     public class HttpTriggerCSharp
     {
-        private readonly ILogger _logger;
+        private readonly ILogger<HttpTriggerCSharp> _logger;
 
-        public HttpTriggerCSharp(ILoggerFactory loggerFactory)
+        public HttpTriggerCSharp(ILogger<HttpTriggerCSharp> logger)
         {
-            _logger = loggerFactory.CreateLogger<HttpTriggerCSharp>();
+            _logger = logger;
         }
 
         [Function("HttpTriggerCSharp")]
+#if NetCore
+        public IActionResult Run([HttpTrigger(AuthorizationLevel.AuthLevelValue, "get", "post")] HttpRequest req)
+        {
+            return new OkObjectResult("Welcome to Azure Functions!");
```

Comment:
The logger isn't used in the NetCore path. We should make sure to include that,
```suggestion
            _logger.LogInformation("C# HTTP trigger function processed a request.");

            return new OkObjectResult("Welcome to Azure Functions!");
```